doc/start/os-recommendations.rst

```diff
@@ -41,6 +41,24 @@ platforms.  Generally speaking, there is very little dependence on
 specific distributions aside from the kernel and system initialization
 package (i.e., sysvinit, systemd).
 
+Quincy
+------
+
++----------+----------+--------------------+--------------+---------+------------+
+| Distro   | Release  | Code Name          | Kernel       | Notes   | Testing    |
++==========+==========+====================+==============+=========+============+
+| CentOS   | 8        | N/A                | linux-4.18   |         | B, I, C    |
++----------+----------+--------------------+--------------+---------+------------+
```

Comment:
@djgalloway can help with a full list of what we're building for quincy + pacific.

We're testing both releases on centos stream 8, rhel 8, and ubuntu 20.04

Reply:
Here's what we *build* for (not necessarily support)
```
Octopus (15.X.X): focal bionic centos7 centos8 buster
Pacific (16.X.X): focal bionic centos8 buster bullseye
Quincy (17.X.X): focal centos8 bullseye
```